snap/snapcraft.yaml

```diff
@@ -310,6 +318,21 @@ parts:
         git describe --tags > ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/edge-proxy.VERSION
       organize:
         bin/edge-proxy: wigwag/system/bin/edge-proxy
+    pe-terminal:
+      plugin: go
+      source: https://github.com/PelionIoT/pe-terminal.git
+      source-commit: 8d2a020eb0bccf40cd9d9e54188e29df531bc019
+      after: [edge-proxy]
```

Comment:
What is the intent here?  Is it really required for edge-proxy to be *built* before pe-terminal can be built? https://snapcraft.io/docs/snapcraft-parts-metadata

Reply:
I think I might have accidentally added this in build section. I was making sure that `pe-terminal` only works if `edge-proxy` is available. Will remove.